cockpit/src/subscriptions-client.js

```diff
@@ -18,6 +18,13 @@
  */
 
 var cockpit = require("cockpit");
+var service = cockpit.dbus('com.redhat.RHSM1', {'superuser': 'require'});
```

Comment:
Why var's instead of const?  I think most of all these will be fixed (ie, you wont assign it to something else later).  And  const in es2015 will still let you mutate the contents of references (just not the reference itself).  In C terms, I tend to think of const like a const pointer, but not a pointer to a const.

```javascript
const names = [];
names.push("sean");  // works
names = []; // error
```